### ISSUE TITLE
Increase the size of the JWT Encoder/Decoder payload textarea

### ISSUE THREAD

**Issue by HBTeun** [DevToys-app/DevToys#1275]
### What improvement do you think would an existing feature or tool in DevToys?

Increase the size of the JWT Encoder/Decoder payload panel. For me it only shows 5 lines and I can't increase the textarea size myself.

### Solution/Idea

Maybe autosize to content? Or just make it show at least 20 lines

**Comment by karkcoder** [comment]
Trying to contribute here. So if this untriaged, I need to wait out for this enhancement to be triaged and approved OR can I just fix this and create a PR for project leads to look at??

**Comment by hahn-kev** [comment]
I'd like to get this fixed too, I'd be happy to contribute, same question as above

**Comment by veler** [comment]
> I'd like to get this fixed too, I'd be happy to contribute, same question as above

@btiteux is working on this :)

**Comment by ilya-chumakov** [comment]
Hi guys, are there any updates on this? This is what it looks like on a QHD screen, I had to rollback to 1.x version because of this issue:

![Image](https://github.com/user-attachments/assets/5f8b53db-2145-414e-a138-4a3268b273d3)

**Comment by roErlend** [comment]
Any updates on this?

**Comment by karkcoder** [comment]
nope, no luck so far, the whole hiding and unhiding of the multiline textbox is not letting auto-sizing (which should expand multiline box similar to the gzip one). I am getting some random result even after some conversion.

<img width="3196" height="1897" alt="Image" src="https://github.com/user-attachments/assets/56884e6e-87b7-400f-b1ca-298629a9c8fe" />

<img width="1389" height="351" alt="Image" src="https://github.com/user-attachments/assets/f3c355e5-5fd2-442e-b236-81fe7302dac4" />

link to layout: https://devtoys.app/doc/articles/extension-development/guidelines/UX/layout/split-grid.html

**Comment by btiteux** [comment]
Hello,

I was a bit off lately sorry about that.

I worked on it and I have the following result:

<img width="1720" height="944" alt="Image" src="https://github.com/user-attachments/assets/52b6ab70-0a45-44c0-90b2-b3aac16836c9" />

<img width="1720" height="944" alt="Image" src="https://github.com/user-attachments/assets/327706b4-5edf-4bbb-bcf2-8a3605f8beb7" />

I'm going to create a pull request for these changes.

**Comment by karkcoder** [comment]
Sounds good. Curious how you fixed the issue with MultiLineText.

**Comment by btiteux** [comment]
@karkcoder the pull request for the changes if you want to look https://github.com/DevToys-app/DevToys.Tools/pull/72

**Comment by btiteux** [comment]
Hello, I'm closing this issue. I merged the pull request to fix this bug.
